README.md

```diff
@@ -182,15 +189,37 @@ function handlerFunction() {
 }
 ```
 
+#### Action matching with blocks
```

Comment:
newline after this heading for readability (i know the rendered output is the same but for source its a little easier on the eyes)